onfido-java/src/test/java/com/onfido/integration/CheckManagerTest.java

```diff
@@ -97,4 +98,22 @@ public void resumeCheck() throws Exception {
     RecordedRequest request = server.takeRequest();
     assertEquals("/checks/id/resume", request.getPath());
   }
+
+  @Test
+  public void downloadCheck() throws Exception {
+    MockWebServer server = mockFileRequestResponse("test", "application/pdf");
+
+    Onfido onfido =
+        Onfido.builder().apiToken("token").unknownApiUrl(server.url("/").toString()).build();
+
+    FileDownload download = onfido.check.download("check id");
```

Comment:
Minor: I'd use `check_id` for this so we can test the exact same id gets used in the path (without the %20)